buildenv/jenkins/jobs/pipelines/Pipeline-OMR-Acceptance

```diff
@@ -80,6 +80,15 @@ timeout(time: 8, unit: 'HOURS') {
                  }
             }
 
+            // Note: we are doing this as a special case rather than include it in SPECS,
+            // because at the moment cmake builds are only supported on java 9.
+            // Also we dont want to bother running tests at the moment.
+            def SHAS = [:]
+            SHAS['OPENJ9'] = ALL_SHAS['OPENJ9']
+            SHAS['OMR'] = ALL_SHAS['OMR']
+            SHAS['OPENJDK'] = buildFile.get_sha(OPENJDK_REPOS['9'].repo, OPENJDK_REPOS['9'].branch)
+            JOBS["Build-TestJDK9-linux_x86-64_cmake"] == { buildFile.workflow(9, linux_x86-64_cmprssptrs, SHAS, OPENJDK_REPOS['9'].repo, OPENJDK_REPOS['9'].branch, OPENJ9_REPO, OPENJ9_BRANCH, OMR_REPO, OMR_BRANCH, 'none') }
```

Comment:
Is it worth defining a var for `VERSION=9`? Since you will be changing it to 10.